pkg/minikube/config/types.go

```diff
@@ -81,6 +81,7 @@ type ClusterConfig struct {
 	ListenAddress           string   // Only used by the docker and podman driver
 	Network                 string   // only used by docker driver
 	MultiNodeRequested      bool
+	DockerEnvInUse          bool
```

Comment:
This should not be part of config. Because in one terminal it might be in use and other terminal not but config is global. So let's remove it

Reply:
@medyagh, without the config change. can you help me with how we can implement this for podman?

for docker, we can make use of this `MINIKUBE_ACTIVE_DOCKERD` env variable at runtime to check if it is the same as the cluster name.

Reply:
For podman u could see the podman-env.go file

But also lets make sure the minikbue Config does not have that DockerEnvInUse ...